### DISCUSSION TITLE
Translate fields title in the Control Panel

### DISCUSSION THREAD

**Discussion by alanpilloud** [statamic/cms#5821] [General]
Hi,

In the control panel, I need to be able to translate the fields title, help texts, etc. in order to provide a better user experience to editors.

Is there a way to achieve this ? If not, we could submit a PR if you provide some guidance on how it would be good to do this.

Thanks a lot !

**Comment by jasonvarga** [comment]
See https://github.com/statamic/ideas/issues/50

**Comment by tejar37** [comment]
The Translated Name / Fields [sys_translated] table stores translated values for text fields where the field type is translated_field (see the dictionary entry)[.](https://www.tejar.pk/computers-accessories/laptops/macbooks)

This option is available for text fields up to 255 characters in length. Some examples are names, titles, and short descriptions. The main fields for this table are:
Table: name of the table this translation applies to.
Element: name of the field this translation applies to.
Label (translate): translated text that users see on forms and lists.
Language: two-character ISO language code for this translated text.
Value: English value that causes this translated text to be displayed.

**Reply by tejar37** [reply]
Go to the Searchanise control panel > Translations section > Instant Search Widget tab.
Type the phrases you[’](https://www.tejar.pk/apple-airpods-pro)d like to display instead of the original ones into the corresponding fields. If you leave a field empty, the text won’t be shown in the widget.
Apply the changes.
